src/layout/ThemeContext.tsx

```diff
@@ -23,19 +25,32 @@ const lightTheme = createTheme({
 })
 
 const { defaultAlgorithm, darkAlgorithm } = theme
+
 export const ThemeProvider: FC<PropsWithChildren> = ({ children }) => {
-  const [isDarkTheme, setIsDarkTheme] = useState(false)
-  const { i18n } = useTranslation()
+  const { setItem, getItem } = useLocalStorage('isDarkTheme')
+  const [isDarkTheme, setIsDarkTheme] = useState(() => {
+    return getItem() === 'true' || false
```

Comment:
i wonder -
if we change `return getItem() === 'true' || false`
to `return getItem()`
will it still work?

Reply:
or even simplier: `  const [isDarkTheme, setIsDarkTheme] = useState(getItem())`

Reply:
Ahh yeah, that's correct it can be simplified as such

Reply:
one more thought i have now: do we have to convert the isDarkTheme value to string before storgin it at localStorage?

Reply:
that is an interesting question @shootermv, I was converting that to string because i think local storage stores both it's values and keys as strings, so i wanted to maintain that flow of data. i could be wrong though. what do you think?